Integrations/python/deephaven_legacy/tutorials/crypto_trades.py

```diff
@@ -40,7 +40,7 @@ def ticking_crypto_milliseconds(interval: int):
     return t
 
 def ticking_crypto_milliseconds_v2(interval: int):
-    from deephaven2 import time_table
+    from deephaven import time_table
```

Comment:
If this is legacy, isn't the method name `timeTable`?

```suggestion
    from deephaven import timeTable
```

Reply:
I don't think this is legacy so it's supposed to be `time_table`

This code works when I manually make the update on the server

```
from deephaven_legacy.tutorials.crypto_trades import *

crypto_streaming = ticking_crypto_milliseconds_v2(25)
```

Reply:
Without the manual update it fails with the following error message

```
Type: <class 'ModuleNotFoundError'>
Value: No module named 'deephaven2'
```